Code/GraphMol/DetermineBonds/DetermineBonds.cpp

```diff
@@ -0,0 +1,62 @@
+//
+//  Copyright (C) 2022 Sreya Gogineni and other RDKit contributors
+//
+//   @@ All Rights Reserved @@
+//  This file is part of the RDKit.
+//  The contents are covered by the terms of the BSD license
+//  which is included in the file license.txt, found at the root
+//  of the RDKit source tree.
+//
+
+#include "DetermineBonds.h"
+#include <GraphMol/RDKitBase.h>
+//#include "EHTTools.h"
+#include <iostream>
+
+namespace RDKit {
+
+
+void determineConnectivity(RWMol &mol, bool useHuckel, int charge, double covFactor) { // accept reference
+    unsigned int numAtoms = mol.getNumAtoms();
+    for (unsigned int i = 0; i < numAtoms; i++) {
+        for (unsigned int j = i + 1; j < numAtoms; j++) {
+            mol.removeBond(i, j);
+            mol.getAtomWithIdx(i)->setNoImplicit(false);
+            mol.getAtomWithIdx(j)->setNoImplicit(false);
```

Comment:
I think you want to do `setNoImplicit(true)` here so that no implicit Hs are found for the atoms.